src/DeepNgToDo/Tests/Frontend/angular/services/DeepNgLoginService.spec.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+import moduleName from '../../../../Frontend/js/app/angular/name';
+
+
+describe('Services', function() {
+  let deepNgLoginService;
+  let $q;
+
+  beforeEach(function() {
+    // load the module.
+    module(moduleName);
+
+    // inject your service for testing.
+    // The _underscores_ are a convenience thing
+    // so you can have your variable name be the
+    // same as your injected service.
+    inject(function(_deepNgLoginService_, _$q_) {
+      deepNgLoginService = _deepNgLoginService_;
+      $q = _$q_;
+    });
+  });
+
+  it('Check constructor sets valid default values', function() {
+    expect(angular.isFunction(deepNgLoginService.$q)).toBe(true);
+  });
+
+
+
+  it('Check anonymousLogin() method with title returns resolved promise with token', function() {
+    let error = null;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).